src/components/Tooltip/Tooltip.js

```diff
@@ -115,7 +117,14 @@ export const Tooltip = ({
                 {children}
               </div>
             ) : (
-              <div className={styles.icon} ref={ref} {...referenceProps}>
+              <div
+                role="button"
```

Comment:
The role is required for this to be seen as interactive and therefore allow the user to tab through the page and open the tooltip, divs shouldn't have a tabIndex on them unless you specify that they are in fact an interactive element, in this case the div is acting like a button, so this is the appropriate role.